crypto/curve25519/curve25519.c

```diff
@@ -4520,13 +4521,9 @@ void GFp_x25519_sc_muladd(uint8_t *s, const uint8_t *a, const uint8_t *b,
   s[31] = (uint8_t)(s11 >> 17);
 }
 
-
-/* Prototype to avoid -Wmissing-prototypes warnings. */
-void GFp_ed25519_scalar_mask(uint8_t a[32]);
-
-void GFp_ed25519_scalar_mask(uint8_t a[32]) {
+void GFp_x25519_scalar_mask(uint8_t a[32]) {
```

Comment:
I think this is just an artifact of the `x25519_scalar_mask()` function only being compiled in some cases, whereas the Ed25519 one was already _unconditionally_ compiled.

The reason I kept the impl with `&= 127` is because it most closely tracked the language in RFC 8032 (step 2 [here](https://tools.ietf.org/html/rfc8032#section-5.1.5)) and the code in RFC 7748 (the `decodeScalar25519` function [here](https://tools.ietf.org/html/rfc7748#section-5)).

Anyway, that was my reasoning. Would you still like to change it to use the `&= 63`, or was that not what you meant?